tony-core/src/main/java/com/linkedin/tony/models/JobMetadata.java

```diff
@@ -37,7 +41,8 @@ public static JobMetadata newInstance(String histFileName) {
     long completed = Long.parseLong(metadata[2]);
     String user = metadata[3];
     String status = metadata[4];
-    return new JobMetadata(id, jobLink, configLink, started, completed, status, user);
+    String rmLink = Utils.buildRMUrl(new YarnConfiguration(), id);
```

Comment:
Instead of creating a `new YarnConfiguration` each time `JobMetadata.newInstance()` is called, can we just pass in a `Configuration` object to this method instead?

Reply:
Sure. Also, I'm assuming that this is to limit the number of `YarnConfiguration` objects created?

Reply:
Yes, and for consistency. In case we want to add something to the `YarnConfiguration` in all of THS, we should be able to do it in one place rather than have to do it in every place we create a `new YarnConfiguration` instance.